<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kirjauduttusisaan</class>
 <widget class="QWidget" name="kirjauduttusisaan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nosta rahaa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn20eur">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>20 $</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn40eur">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>40 $</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn50eur">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>50 $</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn100eur">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>100 $</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnmuuSumma">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>muu summa</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditmuu">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kyselyt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaldokys">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Saldokysely</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTapahtumakys">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tapahtumakysely</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtnKeskeyta">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Keskeyt√§</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditTapahtuma">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>231</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditSaldo">
   <property name="geometry">
    <rect>
     <x>373</x>
     <y>170</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
